### PR TITLE
Docs: add save and load model docs.

### PR DESCRIPTION
**Pull request by lalalapotter**
## Description
This PR is to add save and load model quick-start documents.

### CODE REVIEW

**Review by sgwhat — APPROVED** [review]

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md [hunk 1]
```diff
@@ -118,6 +118,35 @@ est.shutdown()
 print(stats)
 ```
 
+### **Step5: Save and Load the Model**
+
+Orca TF2 Estimator supports 2 formats to save and load the entire model (**TensorFlow SavedModel and Keras H5 Format**). The recommended format is SavedModel, it is the default when you use `estimator.save()`.
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:123]: 2 -> two
it is the default when you use `estimator.save()`. -> which is the default format when ...

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md [hunk 2]
```diff
@@ -118,6 +118,35 @@ est.shutdown()
...
+
+You could also switch the saving format to Keras H5 by:
+
+- Passing `save_format='h5'` to `estimator.save()`.
+- Passing a filename that ends in `.h5` or `.keras` to `estimator.save()`.
+
+**1. SaveModel Format**
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:130]: Saved

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md [hunk 3]
```diff
@@ -118,6 +118,35 @@ est.shutdown()
...
+
+```python
+# save model in SaveModel format
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:133]: saved

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md [hunk 4]
```diff
@@ -118,6 +118,35 @@ est.shutdown()
...
+**2. HDF5 format**
+
+```python
+# save model in H5 format
+estimator.save("/tmp/cifar10_model.h5", save_format='h5')
+
+# load model
+estimator.load("/tmp/cifar10_model.h5")
+```
+
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:149]: Need to add a remark to say something like: if run on yarn, the path should be HDFS

**Comment by hkvision** [comment]
Also need to modify the notebook to add save and load correspondingly.

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md (hunk 1 above)
Comment (lalalapotter) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md]: fixed.

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md (hunk 2 above)
Comment (lalalapotter) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md]: fixed.

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md (hunk 3 above)
Comment (lalalapotter) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md]: fixed.

**Comment by lalalapotter** [comment]
> Also need to modify the notebook to add save and load correspondingly.
fixed

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md [hunk 5]
```diff
@@ -118,6 +118,34 @@ est.shutdown()
 print(stats)
 ```
 
+### **Step5: Save and Load the Model**
+
+Orca TF2 Estimator supports two formats to save and load the entire model (**TensorFlow SavedModel and Keras H5 Format**). The recommended format is SavedModel, which is the default when you use `estimator.save()`.
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:123]: the default `format`

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md [hunk 6]
```diff
@@ -118,6 +118,34 @@ est.shutdown()
...
+
+You could also switch the saving format to Keras H5 by passing `save_format='h5'` or a filename that ends in `.h5` or `.keras` to `estimator.save()`.
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:125]: You could also save to Keras H5 format by ...

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md [hunk 7]
```diff
@@ -118,6 +118,34 @@ est.shutdown()
...
+
+**Noted that, if you use Yarn as your application master, you should save the model to HDFS and load it from HDFS as well.**
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:127]: Note that if you run on Apache Hadoop/YARN cluster, you are recommended to save the model to HDFS and load from HDFS as well.

**Review by hkvision — APPROVED** [review]

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md (hunk 6 above)
Comment (sgwhat) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:125]: save the model (we need to have an object after "save"), or just keep the original sentence?

Comment (hkvision) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md:125]: You could also save the model to Keras H5 format by ...

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md (hunk 5 above)
Comment (lalalapotter) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md]: fixed.

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md (hunk 6 above)
Comment (lalalapotter) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md]: fixed.

File: docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md (hunk 7 above)
Comment (lalalapotter) [reply docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md]: fixed.

**Comment by hkvision** [comment]
LGTM

**Pull request merged**